Iterators.md

```diff
@@ -0,0 +1,176 @@
+# Iterators
+
+### Page 0: Introduction
+
+This section of tutorial will be focused on iterators. If you are unfamiliar with iterators in Python, you might find it helpful to read through the Python tutorial on iterators and generators, which you can find [here](https://docs.Python.org/3/tutorial/classes.html#iterators). This tutorial also references the Python itertools module, the documentation for which can be found [here](https://docs.Python.org/2/library/itertools.html).
```

Comment:
I still think that this section would benefit from having a short line that actually explains *what* an iterator is. It probably isn't as commonly found in Python, but especially considering that they are the first thing that our audience sees, we probably shouldn't immediately redirect them somewhere else (we can briefly explain what it does or how it helps in the intro part instead)?